controllers/vaultstaticsecret_controller.go

```diff
@@ -183,36 +152,10 @@ func (r *VaultStaticSecretReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}, nil
 }
 
-// SetupWithManager sets up the controller with the Manager.
+// makeK8sSecret returns Kubernetes Secret data from an api.KVSecret
+// Deprecated: use vault.MarshalKVData instead
```

Comment:
Do we need to keep this around?

Reply:
Dropped in 7fa55d5ffeda41e8c2705cc32de3385a04e0ce1e